idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -22,7 +39,7 @@ struct VersionInfo {
 }
 
 service Meta {
-    HealthStatus health()
+    HealthStatus health(1: HealthRequest hr)
```

Comment:
Adding new parameters should be backwards compatible since parameters are optional.

cc @abhinav to confirm.